README.rst

```diff
@@ -101,13 +101,27 @@ Design Pattern followed by packages
 
 All tools in edx_django_utils should expose their public api in their __init__.py files. This entails adding to __init__.py all functions/classes/constants/objects that are intended to be used by users of library.
 
-License
--------
+Getting Help
+------------
 
-The code in this repository is licensed under the Apache License, Version 2.0, unless
-otherwise noted.
+If you're having trouble, we have discussion forums at
+`discuss.openedx.org <https://discuss.openedx.org>`_ where you can connect with others in the
+community.
 
-Please see ``LICENSE.txt`` for details.
+Our real-time conversations are on Slack. You can request a `Slack
+invitation`_, then join our `community Slack workspace`_.
+
+For anything non-trivial, the best path is to `open an issue`__ in this
+repository with as many details about the issue you are facing as you
+can provide.
+
+__ https://github.com/openedx/django-config-models /issues
```

Comment:
I think there's an extra space in this URL?

Reply:
[fixed]